src/vnm_mad/remotes/dummy/clean

```diff
@@ -16,4 +16,8 @@
 # limitations under the License.                                             #
 #--------------------------------------------------------------------------- #
 
-exit 0
+stdin="$(</dev/stdin)"
+
+for f in "$0".d/* ; do
+  echo "$stdin" | ./"$f" "$@"  || exit 1
```

Comment:
Probably missing
```
[ -x $f ] && echo ....
```

Reply:
Done, I added `|| exit 0`, else it would run exit 1